packages/frontend-test-utils/package.json

```diff
@@ -36,7 +36,9 @@
     "@backstage/frontend-plugin-api": "workspace:^",
     "@backstage/plugin-app": "workspace:^",
     "@backstage/test-utils": "workspace:^",
-    "@backstage/types": "workspace:^"
+    "@backstage/types": "workspace:^",
+    "@backstage/version-bridge": "workspace:^",
+    "zod": "^3.22.4"
```

Comment:
This probably isn't handled gracefully by knip. But that one's not really a serious thing to worry about at this point :)

Also mild source of confusion / inconvenience to have "viral" dependencies in the internal packages that spiral outward. But I mean, there's really no good way around that, so 👍

Reply:
Yep and in general it's likely important to keep dependencies in internal packages to a minimum